package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
no it is not included. That's why it does not work on pypi since the information like version author cannot be read